test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_smtrcomp.py

```diff
@@ -166,7 +166,7 @@ def test_horovod(self, sagemaker_session, ecr_image, framework_version, instance
         captured = capsys.readouterr()
         _assert_training_compiler_invoked(captured)
 
-
+    @pytest.mark.usefixtures("feature_smdp_present")
```

Comment:
Please leave 2 lines of space before the start of a function.

```suggestion

    @pytest.mark.usefixtures("feature_smdp_present")
```